src/nvim/tui/tui.c

```diff
@@ -1654,7 +1657,8 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
   }
 
   // Blacklist of terminals that cannot be trusted to report DECSCUSR support.
-  if (!(st || (vte_version != 0 && vte_version < 3900) || konsole)) {
+  if (!(st || (vte_version != 0 && vte_version < 3900)
+        || (konsolev > 0 && konsolev < 180770))) {
```

Comment:
@ahmadsamir is this how the version should be checked?

Reply:
Yes, exactly. (And sorry for not replying earlier :)).